lib/attach-middleware.js

```diff
@@ -13,12 +13,17 @@ function logError(err, req, res, next) {
   next(err);
 }
 
-function fixFilePaths(coverageData, fileLookup) {
-  coverageData.path = fileLookup[coverageData.path];
+function fixFilePaths(coverageData, fileLookup, configPath) {
```

Comment:
So the addon needs the path of the source file so it can render an html file of the source to show line coverage.  Since we are using a monorepo the source for the addons will be a directory back from the app.  Ideally, we'd be able to access the source from the addon via node modules so you can get coverage on 3rd party addons.  Looking for a way to do that still.